client/blocks/importer/util.ts

```diff
@@ -1,12 +1,10 @@
 import { FEATURE_UPLOAD_THEMES_PLUGINS, planHasFeature } from '@automattic/calypso-products';
-import { get } from 'lodash';
-import { SitesItem } from 'calypso/state/selectors/get-sites-items';
-import { Importer } from './types';
+import type { Importer } from './types';
+import type { SiteDetails } from '@automattic/data-stores';
 
 export const getImporterTypeForEngine = ( engine: Importer ) => `importer-type-${ engine }`;
 
-export function isTargetSitePlanCompatible( targetSite: SitesItem ) {
-	const planSlug = get( targetSite, 'plan.product_slug' );
-
+export function isTargetSitePlanCompatible( targetSite: SiteDetails | undefined ) {
+	const planSlug = targetSite?.plan?.product_slug;
```

Comment:
This replaces lodash `get()` with pure JS. The functionality should be the same.